mw-webapp/src/component/footer/Footer.module.scss

```diff
@@ -1,36 +1,44 @@
 @import "src/styles/variables";
 
-$userNameFontSize: 16px;
+$logoHeight: 55px;
+$privacyPaddingTop: 10px;
+$privacyPaddingBottom: 12px;
+$iconSize: 32px;
+$privacyLineHeight: 140%;
 
 .footer {
   background-color: var(--quaternaryBgColor);
+  margin-inline: $paddingMedium;
 }
 
 .footerWrapper {
   width: 100%;
   max-width: $maxWidthContainer;
+  align-items: flex-start;
   justify-content: space-between;
-  padding: $paddingBig $paddingMedium;
   border-bottom: $primaryBorder solid var(--secondaryStrokeColor);
   margin: 0 auto;
+  padding-block: $paddingBig;
```

Comment:
I understand that  padding-block and margin-inline are more modern properties and that set values depends on text oriented.  But if we will use this properties we should rewrite all styles for these properties/ Otherwise some block could be broken.  What do you think?

Reply:
This line is related to the bug (I mention in [other](https://github.com/tritonJS826/masters-way/pull/1152#discussion_r1651456526) reply), so I will try to rewrite it without padding-block.

Reply:
'inline' and 'block' are gone